src/awst_build/arc4-util.ts

```diff
@@ -108,3 +113,18 @@ export function ptypeToArc4EncodedType(ptype: PType, sourceLocation: SourceLocat
 
   throw new CodeError(`${ptype} cannot be encoded to an ARC4 type`, { sourceLocation })
 }
+
+export function zeroValue(ptype: ARC4EncodedType, sourceLocation: SourceLocation): Expression {
+  return nodeFactory.intrinsicCall({
+    opCode: 'bzero',
+    immediates: [],
+    stackArgs: [
+      nodeFactory.uInt64Constant({
+        value: ARC4EncodedType.bitsToBytes(ptype.minBitSize),
```

Comment:
Does this work for more complex nested types - like a tuple with dynamic elements where the header needs offsets?